activemq-client/src/main/java/org/apache/activemq/ActiveMQConnection.java

```diff
@@ -307,6 +307,44 @@ public JMSConnectionStatsImpl getConnectionStats() {
     /**
      * Creates a <CODE>Session</CODE> object.
      *
+     * @throws JMSException if the <CODE>Connection</CODE> object fails to
+     *                 create a session due to some internal error or lack of
+     *                 support for the specific transaction and acknowledgement
+     *                 mode.
+     * @since 2.0
+     */
+	@Override
```

Comment:
This is totally a non-functional style comment, but I feel like every new method you've added has a different indentation level 😅

Reply:
IDE preferences didn't come over. Fixed in latest commit

Reply:
fixed